### PR TITLE
initial try to implement the prcp_tot indice

### PR DESCRIPTION
**Pull request by Zeitsperre**

### CODE REVIEW

File: tests/test_indices.py [hunk 1]
```diff
@@ -103,7 +103,7 @@ class Test_prcptotal():
     def test_yearly(self):
         da_std = self.da_std
         out_std = xci.prcptot(da_std, units='mm')
-        l_years = np.unique(da_std.time.dt.year)
+        # l_years = np.unique(da_std.time.dt.year) TODO: Unused local variables are a PEP8 violation
```
Comment (Zeitsperre) [inline tests/test_indices.py]: PEP8 convention F841: Should we consider easing up on this violation?

**Comment by Balinus** [comment]
I suggest `prcp_tot` to keep inline with xclim usage of naming functions. e.g. see `summer_days`

File: xclim/indices.py [hunk 2]
```diff
@@ -449,11 +454,70 @@ def tn_min(tasmin, freq='YS'):
...
+
+    """
+    # TODO deal with the time_boundaries
+
+    # resample the precipitation to the wanted frequency
+    arr = pr.resample(time=freq)
+    # cumulate the values over the season
+    output = arr.sum(dim='time')
+    # unit conversion as needed
+    if units == 'kg m-2 s-1':
```
Comment (tlogan2000) [inline xclim/indices.py]: NRCAN data has 'mm s-1'

**Comment by huard** [comment]
I propose we do the unit conversion in a unit checking decorator.

Le mer. 26 sept. 2018 10:07, Travis Logan <notifications@github.com> a
écrit :

**Pull request merged**